src/Sylius/Bundle/CoreBundle/Generator/ProductVariantGenerator.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Generator;
+
+use Sylius\Bundle\ProductBundle\Generator\VariantGenerator;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Sylius\Component\Variation\Model\VariableInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class ProductVariantGenerator extends VariantGenerator
```

Comment:
Unfortunately we need for now, as price is required to persist variant ;/  However, after fixes from [this PR](https://github.com/Sylius/Sylius/pull/5059) it indeed won't be necessary ;)